features/step_definitions/admin_can_login_steps.rb

```diff
@@ -0,0 +1,17 @@
+Given("The following user exists") do |table|
+ table.hashes.each do |user|
+   FactoryBot.create(:user, user)
+ end
+end
+
+Given("I am on the Admin login page") do
+ visit new_admin_user_session_path
+end
+
+Then("I click submit") do
+ find_button('Login').click
+end
+
+Then("I should be redirected to admin dashboard") do
+ visit admin_root_path
+end
```

Comment:
Same here! 
We already have these some of these steps!

Reply:
This is admin root not normal root